pkg/api/routes.go

```diff
@@ -55,7 +55,8 @@ func NewRouteHandler(c *Controller) *RouteHandler {
 func (rh *RouteHandler) SetupRoutes() {
 	rh.c.Router.Use(AuthHandler(rh.c))
 
```

Comment:
Add a comment here saying that "authz is being enabled because authn is found"

Reply:
Also make sure "zot verify" catches this situation.

Reply:
I already added 2 test cases for this: https://github.com/anuvu/zot/pull/221/files#diff-c3e95b9d0ad189ba68bf30694e60fd679eb406ac541ff635f3171f18a6d794a4R88

Or do you mean another situation?